### PR TITLE
[CS/Docs] Update readme

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Delete manual package install requirements
- Update demo footage

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

File: Applications/Tizen_native/CustomShortcut/README.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+
 
 `CustomShortcut` is a nntrainer demo app that maps user defined symbol to smily face or sad face.
 
 For example, user can draw `^^` few times to train a model to label the drawing as 😊.
 
 Built and tested at `tizen studio 3.7` with `wearable-6.0 profile`
 
-tested on `SM-R800` (it should run in other wearable but it is not guaranteed)
+tested on `SM-R800` (it should run in other wearable but it is not tested)
```
Comment (kparichay) [inline Applications/Tizen_native/CustomShortcut/README.md:14]: it is not tested -> it has been verified
it should run ~in~ on other wearable

**Review by kparichay — APPROVED** [review]
LGTM

**Comment by zhoonit** [comment]
Updated.

**Comment by zhoonit** [comment]
Done

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM 👍🏼👍🏼👍🏼

**Pull request merged**